src/current/v26.1/manage-logical-data-replication.md

```diff
@@ -80,6 +74,80 @@ CONSTRAINT dlq_113_public_promo_codes_pkey PRIMARY KEY (ingestion_job_id ASC, dl
 )
 ~~~
 
+#### Manage entries in the DLQ
+
+LDR does not pause when writes are sent to the DLQ. You must manage the DLQ manually by examining each entry in the DLQ and either reinserting the entry or deleting it from the DLQ. If you have multiple DLQ entries, manage them in order from most recent to least recent. 
+
+To manage an entry in the DLQ:
+
+1. In the destination database's DLQ table, examine the `incoming_row` column to find the primary key and values for the entry.
+    
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    #On the destination database:
+    SELECT id, dlq_timestamp, incoming_row FROM crdb_replication.dlq_271_foo;
+    ~~~
+
+    In this example result, `incoming_row` contains a primary key of `207` identified by the column `my_id`, as well as the values of the entry's columns `created_at` and `payload`.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    id | dlq_timestamp | incoming_row
+    ----------------------+---------------------+----------+-------------------------------+-----------------------------------------------------------------
+    106677386757203 | 2025-04-25 25:32:28.435439+00 | {"created_at": "2025-04-25:35:00.499499", "payload": "updated_value", "my_id": 207}
+    ~~~
+
+1. Determine whether the values for the entry in the DLQ match the values for the entry in the destination table and the source table respectively:
+
+    1. On the destination database, check the values for the entry and the replicated time:
+
+        {% include_cached copy-clipboard.html %}
+        ~~~ sql
+        #On the destination database:
+        SELECT * FROM destDB.foo WHERE my_id = 207;
+
+        WITH t as (SHOW LOGICAL REPLICATION JOBS)                                                                        
```

Comment:
nit: if the user is running multiple LDR jobs, they should grab the replicated time for the relevant LDR job, so  perhaps we should modify the query to return the job_id. 

`with t as (SHOW LOGICAL REPLICATION JOBS) SELECT job_id, replicated_time FROM t`